apply-error-prone-suggestions.sh

```diff
@@ -9,13 +9,14 @@
 set -e -u -o pipefail
 
 if [ "${#}" -gt 1 ]; then
-  echo "Usage: ./$(basename "${0}") [PatchChecks]"
+  echo "Usage: ${0} [PatchChecks]"
```

Comment:
We no longer assume that the script is executed from the directory in which it resides.